src/redux/pattern/pattern.sagas.js

```diff
@@ -63,8 +63,6 @@ function* handleAddPattern({ payload }) {
   try {
     yield put(setPatternLoading(true));
     yield call(createPattern, payload);
-    const patterns = yield call(getAllPatterns, null);
-    yield put(setPatterns(patterns));
     yield put(setSnackBarSeverity('success'));
```

Comment:
I think you need here to dispatch action to save pattern to the store

Reply:
it is not necessary to save new pattern in store because immediately after creating new pattern we are redirected to pattern main page. When we go that page new request is made and we receive all pattern including new one